docs/UserGuide.md

```diff
@@ -52,198 +53,292 @@ Tips are helpful suggestions that may improve your experience using TutorAid.
 
-### Step 5: Deleting a student
+### 4.5: Deleting a student
 {:.no_toc}
-Lastly, imagine one of your students `Roy Balakrishnan` has graduated from secondary school and no longer requires your tutoring services. You wish to remove his student details
-since he is no longer your student. To do so, you can easily delete his data from TutorAid by following the steps below:
-1. Type `list` in the command box to see the list of all your students in the Student Panel.
-2. Locate `Roy Balakrishnan` in the Student Panel and take note of his index. In this case, it would be `6`.
-3. Type `del -s 6` into the command box. Upon pressing enter, all details belonging to `Roy Balakrishnan` will be removed.
+Lastly, imagine that your student, Alex Yeoh, has graduated from secondary school and wishes to discontinue his lessons. Thus, would like to remove his student details since he is no longer your student. 
+You can easily delete his data from TutorAid by following the steps below:
+   
+1. Type `del -s 1` into the command box. 
+   
+    > :bulb: The `1` in the command above refers to the index number of Alex in the Student Panel. For more information, you can refer to the detailed documentation for the [Delete a Student](#deleting-a-student-del--s) command.
+   
+1. Upon pressing ENTER, all of Alex's details will be removed from TutorAid. Observe how Alex's details are not found in the Student Panel and that his name has been removed from the student list for the _Maths 1_ in the Lesson Panel:
 
-Now, the Student and Lesson Panels will be updated accordingly without any data of `Roy Balakrishnan`.
+   ![](images/deleteAlex.png)
 
 ### End
 {:.no_toc}
-Congratulations on successfully completing TutorAid's beginner's tutorial :tada: 
-But why quit now? Since you have completed the tutorial, you can go ahead and key in `clear` in the command box to remove all the sample data and to add in some of your own students and lessons. 
-You can also experiment with some of the other TutorAid commands that are listed below in [Features](#5-features). Happy Learning! 
+Congratulations on successfully completing TutorAid's beginner's tutorial :tada: <br>
+
+But why quit now? Since you have completed the tutorial, you can go ahead and key in `clear` in the Command Box to remove all the sample data and to add in some of your own students and lessons.<br>
+
+You can also learn and experiment with some of the other TutorAid commands that we have not covered in the tutorial.<br> 
+You may find the following commands particularly handy:
+* [Deleting a student from a lesson](#deleting-students-from-lessons-del--sl)
+* [Deleting a progress note from a student](#deleting-progress-from-a-student-del--p)
+* [Finding a student](#finding-students-by-name-find--s)
+* [Finding a lesson](#finding-lessons-by-name-find--l)
+
+All the commands available in TutorAid are listed below in [Features](#5-features) section for your reference. Happy Learning! :smile:
 
 ***
 
 # 5. Features
```

Comment:
How about adding a 
> 💡 This section is meant users who have completed the Beginner's Tutorial. It provides a complete documentation of TutorAid's commands.

Reply:
I initially didn't want to phrase it that way because I thought the tutorial is optional after all and users can just dive right into this section if they want, but if you think it would still be better to include this I will!